rpcs3/Input/basic_keyboard_handler.cpp

```diff
@@ -160,139 +160,141 @@ void basic_keyboard_handler::LoadSettings()
 		return;
 	}
 
+	std::vector<KbButton> v_buttons = m_keyboards[0].m_buttons;
```

Comment:
```suggestion
	std::vector<KbButton>& buttons = m_keyboards[0].m_buttons;
```

Reply:
Fix it.
[https://github.com/RPCS3/rpcs3/pull/9346/commits/8a2c31ad994ca2e8b71d60371fadbf2c6137326c](url)